cmds/less/less.go

```diff
@@ -337,12 +337,13 @@ func (s *searchResults) Prev(line int64) (searchResult, bool) {
 	return s.results[i], true
 }
 
-func (l *Lesser) search(s string) searchResults {
+func (l *Lesser) search(s string) *searchResults {
 	reg, err := regexp.Compile(s)
 	if err != nil {
 		// TODO(prattmic): display a better error
 		log.Printf("regexp failed to compile: %v", err)
-		return NewSearchResults()
```

Comment:
Don't change this.

Reply:
Done! I also changed flag to pflag because that is what u-root commands are migrating too.